app/assets/stylesheets/common/foundation/base.scss

```diff
@@ -146,6 +146,10 @@ tr {
   }
 }
 
+.topic-list-item {
+  min-height: 69px; // (70px with 1px border)
```

Comment:
I am hesitant about using a `min-height`value in core, this means that smaller font viewers will get extra space, same for non-standard fonts that have small line heights by default. 

Not sure if there is another way to reach the same goal (which I'm guessing is constant height that for some reason is lacking?).

Reply:
I can look further into it, but constant height wouldn't work, as sometimes the main link / topic link is more than 1 line.

Reply:
I didn't mean to suggest we use a constant height, quite the contrary. I was just was trying to understand the PR, it looks like somewhere the rows have inconsistent height (all else being equal, i.e. title length etc), so the `min-height` is supposed to correct that. But why is the height inconsistent?

Reply:
The heights themselves were not inconsistent. They were just a couple pixels shorter than they are in core at the moment. I assume this is due to how flex works and calculates heights.

In this latest commit, I adjusted the line heights a small bit to account for this, and removed a min-height solution.

Would this be ok with you?